.github/workflows/prettier.yml

```
@@ -0,0 +1,21 @@
+on: [push, workflow_dispatch]
+
+jobs:
+  sort:
+    runs-on: ubuntu-latest
+    name: Prettier
+    if: ${{ github.repository_owner != 'EddieHubCommunity' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Prettier
+        run: |
+          npm ci
+          npx prettier --write public/data/*.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: 'style(data): sort alphabetically'
```

think the commit message should be something like:
```suggestion
          commit_message: 'style: format files'
```